apps/andi/README.md

```diff
@@ -8,16 +8,26 @@ Interactions with Redis are abstracted with `smartcitiesdata.smart_city*` functi
 
 ## Running Andi
 
+#### Prerequisites for auth0 setup: 
+- In `config/integration.exs`, you must enable HTTPS in order to securely access the Auth0 login page. There is a comment block with the necessary config under the `Endpoint` configuration that can be uncommented.
+- You must create self-signed SSL certificates in order for this configuration to take hold. Generate `key.pem` and a `cert.pem` file in the `priv` folder. Instructions for how to generate these keys can be found here: https://devcenter.heroku.com/articles/ssl-certificate-self
```

Comment:
Is it reasonable to provide a script to do this automatically?

Reply:
yeah probs